demo/psa/prov-verif-e2e/README.md

```diff
@@ -17,19 +17,75 @@ end to end PSA Demonstration.
 
 ## Creation of PSA Endorsements
 
-* The Supply Chain Endorsements used in provisioning pipeline setup below are created using the instructions given [here](./COCLI_README.md) Use these instructions if one wants to create their own Endorsements or modify existing ones.
+First create new Concise Module Identifiers (CoMID's) and use them in creating Concise Reference Integrity Manifests (CoRIM's) using reference templates located under `docs/demo/psa/prov-verif-e2e/data/templates` to provision them in Veraison Verification Service.
 
-## Provisioning pipeline setup
+More details about CoRIM and CoMID can be found [here](datatracker.ietf.org/doc/draft-ietf-rats-corim/)
 
-* Install `cocli` tool using following command
+### Initial Setup
 
-In a new shell session
+Do this ONLY if it is not done already:
+
+In a new bourne shell session
+
+```shell
+export TOPDIR=$(pwd)
+```
+
+* Install `cocli` tool using following command
 
 ```shell
 go install github.com/veraison/corim/cocli@demo-psa-1.0.0
 ```
 
-To run provisioning test you need three parallel shell sessions.
+```shell
+git clone https://github.com/veraison/docs
+```
+
+* Remember this shell as shell-1. 
+
+### Create CoMID's
+
+```shell
+cd ${TOPDIR}/docs/demo/psa/prov-verif-e2e
+```
+
+* Create CoMID for Trust Anchors using given JSON template
+
+Please inspect template JSON file `data/templates/comid-psa-iak-pub.json` and modify anything as per your requirement
+
+```shell
+cocli comid create --template=data/templates/comid-psa-iak-pub.json
+```
+
```

Comment:
It's worth adding a line  describing what the expected output of the above command would be. E.g. "This will create a new CoMID in CBOR format in the current directory called `comid-psa-iak-pub.cbor`".

Reply:
I can certainly add this clarification